hack/test-cmd.sh

```diff
@@ -217,22 +218,20 @@ runTests() {
   #######################
 
   # Make sure the UI can be proxied
-  start-proxy --api-prefix=/
+  start-proxy
   check-curl-proxy-code /ui 301
   check-curl-proxy-code /metrics 200
+  check-curl-proxy-code /api/ui 404
   if [[ -n "${version}" ]]; then
     check-curl-proxy-code /api/${version}/namespaces 200
   fi
+  check-curl-proxy-code /healthz/ 200
```

Comment:
Can you modify the proxy invocation so that it doesn't use healthz as the path for this? there's a real /healthz on apiserver, so the fact that this returns 200 doesn't tell us whether it went through the proxy or not...